x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/fielddata_frequency_filter_absolute.tsx

```diff
@@ -4,24 +4,64 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React from 'react';
-import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiFormControlLayoutDelimited,
+  EuiFieldNumber,
+  EuiFieldNumberProps,
+  EuiFormRow,
+} from '@elastic/eui';
 
-import { FieldHook, NumericField } from '../../../shared_imports';
+import { FieldHook } from '../../../shared_imports';
 
 interface Props {
   min: FieldHook;
   max: FieldHook;
 }
 
 export const FielddataFrequencyFilterAbsolute = ({ min, max }: Props) => {
+  const minIsInvalid = !min.isChangingValue && min.errors.length > 0;
+  const minErrorMessage = !min.isChangingValue && min.errors.length ? min.errors[0].message : null;
+
+  const maxIsInvalid = !max.isChangingValue && max.errors.length > 0;
+  const maxErrorMessage = !max.isChangingValue && max.errors.length ? max.errors[0].message : null;
+
   return (
-    <EuiFlexGroup>
-      <EuiFlexItem>
-        <NumericField field={min} />
-      </EuiFlexItem>
-      <EuiFlexItem>
-        <NumericField field={max} />
-      </EuiFlexItem>
-    </EuiFlexGroup>
+    <EuiFormRow
+      fullWidth
+      isInvalid={minIsInvalid || maxIsInvalid}
+      error={minErrorMessage || maxErrorMessage}
+      label={
+        <FormattedMessage
+          id="xpack.idxMgmt.mappingsEditor.fielddata.frequencyFilterAbsoluteFieldLabel"
+          defaultMessage="Min/max frequency absolute"
```

Comment:
I wonder if this would be more human-readable if we changed it to "Absolute frequency range". WDYT?